## Macrosynteny methods for Montmorency genome paper

Used MCscan (Python version) from JCVI.   
 Citation: https://www.science.org/doi/10.1126/science.1153917  
 Link: https://github.com/tanghaibao/jcvi/wiki/MCscan-(Python-version)

Generated .cds and .bed files from the coding sequence and gff files of each reference genome per MCscan tutorial.
Created a list of syntenic blocks between Montmorency and each progenitor genome using 'jcvi.compara.catalog ortholog'. 
Generated karyotype figures using the 'jcvi.graphics.karyotype' command.

Additional image editing (contig lengths, chromosome labeling) done in Inkscape 1.0.1 (https://inkscape.org)
